packages/ui/src/stack/index.tsx

```diff
@@ -7,7 +7,7 @@ export * from './types';
 
 const Stack = forwardRef<any, StackProps>(
   ({ isInline, children, align, justify, spacing = 2, shouldWrapChildren, ...rest }, ref) => {
-    const validChildren = Array.isArray(children) && children.filter(isValidElement);
+    const validChildren = Array.isArray(children) ? children.filter(isValidElement) : [];
```

Comment:
Why was this changed?

Reply:
Because it caused a TS error after I disabled `skipNullChecks`. Technically, before, this would result in `validChildren` being either `false` or `[]`. That was causing errors down the file. With this change, `validChildren` can only be an array.